example/pyproject.toml

```
@@ -1,8 +1,7 @@
 [build-system]
 requires = [
     "wheel",
-    "setuptools>=45",
-    "setuptools_scm[toml]>=6.0",
+    "setuptools",
```

It looks ok to me removing the version pinnings since they change quite frequently, but I'd leave `setuptools_scm` there regardless. We're using this small example as template and I prefer having this package explicitly defined even if not necessary for running the tests on the example.

```suggestion
    # Check the updated version pinning requirements in:
    # https://github.com/pypa/setuptools_scm
    "setuptools",
    "setuptools_scm[toml]",
```